scripts/qa_crowbarsetup.sh

```diff
@@ -1927,7 +1943,13 @@ function onadmin_addupdaterepo()
 {
     pre_hook $FUNCNAME
 
-    local UPR=/srv/tftpboot/repos/Cloud-PTF
+    onadmin_set_source_variables
+
+    if iscloudver 5plus; then
```

Comment:
For my taste, there is too much repetition of this pattern. We should introduce a $repodir variable that points to the right place, so this would remain one line instead of 5:
local UPR=$repodir/Cloud-PTF